src/CNGLOB.H

```diff
@@ -101,6 +101,51 @@ typedef HANDLE HINSTANCE;
 typedef HINSTANCE HMODULE;
 #endif
 
+#if defined(__linux__) || defined(__APPLE__)
```

Comment:
Could you elaborate a bit on how these are used? I didn't quite understand where they come from.

Reply:
They are Windows definition and are primarily used for error reports. They can be found on lines [387](https://github.com/cse-sim/cse/blob/master/src/ENVPAK.CPP#L387-L393) and [423](https://github.com/cse-sim/cse/blob/master/src/ENVPAK.CPP#L423-L427).

Reply:
OK, this looks like something that will need a bit more research. I started [here](https://stackoverflow.com/questions/2219244/dealing-with-floating-point-exceptions). This is one of those places where it'll be important to separate what is platform-specific, vs what is compiler (or C++-library)-specific. I'm not well-versed in floating point exceptions, but I'll do some research as well.

Reply:
OK, this looks like something that will need a bit more research. I started [here](https://stackoverflow.com/questions/2219244/dealing-with-floating-point-exceptions). This is one of those places where it'll be important to separate what is platform-specific, vs what is compiler (or C++-library)-specific. I'm not well-versed in floating point exceptions, but I'll do some research as well.